firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/encoders.kt

```diff
@@ -22,17 +22,3 @@ internal fun <T> encode(strategy: SerializationStrategy<T>, value: T, shouldEnco
         shouldEncodeElementDefault,
         FieldValue.serverTimestamp()
     )
-
```

Comment:
android and ios indeed encode data as map, however JS encodes it to json. updated platform specific code